java/java.editor/src/org/netbeans/modules/java/editor/fold/JavaElementFoldManager.java

```diff
@@ -76,7 +76,11 @@ public class JavaElementFoldManager implements FoldManager {
 	     new org.netbeans.api.editor.fold.FoldTemplate(3, 2, "/**...*/")); // NOI18N
 
     @NbBundle.Messages("FoldType_Methods=Methods")
-    public static final FoldType CODE_BLOCK_FOLD_TYPE = FoldType.MEMBER.derive("method", Bundle.FoldType_Methods(), 
+    public static final FoldType METHOD_BLOCK_FOLD_TYPE = FoldType.MEMBER.derive("method", Bundle.FoldType_Methods(), 
+            new org.netbeans.api.editor.fold.FoldTemplate(1, 1, "{...}")); // NOI18N
+
+    @NbBundle.Messages("FoldType_CodeBlocks=Code Blocks")
```

Comment:
Please also change the reference in `java.editor.lib` - class `org.netbeans.editor.ext.java.JavaFoldManager`. This is a deprecated support for ancient modules that reference symbolically `CODE_BLOCK_FOLD_TYPE` (uses implementation dependency on `java.editor`) - otherwise the old modules would change their folding semantics.

Reply:
I added it as requested. Hope this was the right and only place where I need to add it.